### ISSUE TITLE
WebAPI - No 'Access-Control-Allow-Origin' header is present on the requested resource

### ISSUE THREAD

**Issue by tomerpeled** [aspnet/Home#2018]
I'm encountering an issue with CROS while using **IAsyncResourceFilter** implementation.
I want to be able to call my actions from other domains as well...

I've defined the CORS policy under my Startup file as the following:
  ```
  services.AddCors(options =>
            {
                options.AddPolicy("AllowAllOrigins",
                 builder =>
                 {
                     builder.AllowAnyMethod().AllowAnyHeader().AllowAnyOrigin();
                 });
            });
```
And under the Configure method:
 `app.UseCors("AllowAllOrigins");`

It works fine **without** using a **TypeFilterAttribute** which use **IAsyncResourceFilter**.

For example:
Calling my API action without any TypeFilterAttribute attribute works:
```
public bool Get()
{
 return true;
}
```

But when adding my TypeFilterAttribute as follows it doesn't work and returns the error about the CROS:

```
[MyTypeFilterAttribute("test")]
public bool Get()
{
 return true;
}
```

Anything I'm missing? What should I add when using **IAsyncResourceFilter**?

**Comment by markrendle** [comment]
Can you post the code for `MyTypeFilterAttribute`?

**Comment by tomerpeled** [comment]
Hi,
The issue was fixed, see 43704738/asp-net-core-webapi-no-access-control-allow-origin-header-is-present

I'll close this issue. 

Thanks, 
Tomer
